guides/common/modules/con_lifecycle-status-of-rhel-hosts.adoc

```diff
@@ -0,0 +1,17 @@
+[id="lifecycle_status_of_rhel_hosts_{context}"]
+= Lifecycle Status of RHEL Hosts
+
+{Project} provides multiple mechanisms to display information about upcoming End of Support (EOS) events for your {RHEL} hosts:
+
+* Notification banner
+* A column on the Hosts index page
+* Alert on the Hosts index page for each host that runs {EL} with an upcoming EOS event in a year as well as when support has ended
```

Comment:
```suggestion
* Alert on the Hosts index page for each host that runs {RHEL} with an upcoming EOS event in a year as well as when support has ended
```
You might want to verify this with a SME, but are you sure that the lifecycle status other than Unknown can be displayed for example for CentOS hosts? CentOS is EL as well. If this for RHEL specifically, then let's be exact about it.
The same applies to other occurences.

Reply:
Double checking with SME.

Reply:
Confirmed it's just RHEL.